cmd/course-user-drop/main.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"github.com/alecthomas/kong"
+
+	"github.com/edulinq/autograder/internal/api/core"
+	"github.com/edulinq/autograder/internal/api/courses/users"
+	"github.com/edulinq/autograder/internal/cmd"
+	"github.com/edulinq/autograder/internal/config"
+	"github.com/edulinq/autograder/internal/log"
+)
+
+var args struct {
+	config.ConfigArgs
+	cmd.CommonOptions
+
+	TargetEmail string `help:"Email of the user to drop." arg:""`
+	CourseID    string `help:"ID of the course." arg:""`
+}
+
+func main() {
+	kong.Parse(&args,
+		kong.Description("Drop a user from a course."),
```

Comment:
/s/from a course/from the course

(Match the [python interface](https://github.com/edulinq/autograder-py/blob/9642fa80dd07925f94bed620fe12497524711f32/autograder/api/courses/users/drop.py#L15)).